waspc/cli/Command/Telemetry.hs

```diff
@@ -0,0 +1,32 @@
+module Command.Telemetry
+    ( considerSendingData
+    ) where
+
+import           Control.Monad             (when)
+import           Control.Monad.Except      (catchError, throwError)
+import           Control.Monad.IO.Class    (liftIO)
+import           Data.Maybe                (isJust)
+import qualified System.Environment        as ENV
+
+import           Command                   (Command, CommandError (..))
+import           Command.Telemetry.Common  (ensureTelemetryCacheDirExists)
+import qualified Command.Telemetry.Project as TlmProject
+import qualified Command.Telemetry.User    as TlmUser
+
+-- | Sends telemetry data about the current Wasp project, if conditions are met.
+-- If we are not in the Wasp project at the moment, nothing happens.
+-- If telemetry data was already sent for this project in the last 12 hours, nothing happens.
+-- If env var WASP_TELEMETRY_DISABLE is set, nothing happens.
+considerSendingData :: Command ()
```

Comment:
sounds like a very polite function :D

Reply:
Hehe :D. How would you name it? I thought `sendData` is misleading because in that case I expect it to send it for sure, while it might not send it under some circumstances.